src/components/Sidebar/Sidebar.tsx

```diff
@@ -1,29 +1,120 @@
 import React from 'react';
+import { useState } from 'react'
 import './Sidebar.scss';
 import { GlobalProps } from '../app/App';
+import logo from '../../images/logo.svg'
+import logotext from '../../images/logo-text.svg'
+import feed_normal from '../../images/feed_normal.svg'
+import feed_selected from '../../images/feed_selected.svg'
+import explore_normal from '../../images/search_normal.svg'
+import explore_selected from '../../images/search_selected.svg'
+import notif_normal from '../../images/notification_normal.svg'
+import notif_selected from '../../images/notification_selected.svg'
+import direct_normal from '../../images/direct_normal.svg'
+import direct_selected from '../../images/direct_selected.svg'
+import tv_normal from '../../images/tv_normal.svg'
+import tv_selected from '../../images/tv_selected.svg'
+import stats_normal from '../../images/stats_normal.svg'
+import stats_selected from '../../images/stats_selected.svg'
+import settings_normal from '../../images/settings_normal.svg'
+import settings_selected from '../../images/settings_selected.svg'
 import ProfileInfo from "../ProfileInfo/ProfileInfo";
+import NavItem from "../NavItem/NavItem";
 
-
+export enum Navigation {
+      FEED = 'Feed',
+      EXPLORE = 'Explore',
+      NOTIFICATIONS = 'Notifications',
+      DIRECT = 'Direct',
+      IGTV = 'IG TV',
+      STATS = 'Statistics',
+      SETTINGS = 'Settings'
+}
 
 interface SidebarProps extends GlobalProps {
 
 }
 
-const Sidebar: React.FC<SidebarProps> = (props) => (
-  <div className={`${props?.className} Sidebar bg-gray-dark`}>
-      <div className={'mt-2'}>
-            <img src={'/assets/images/logo.svg'} alt={'Instagram Logo'} className={'w-8 ml-4 sm:mt-1 sm:ml-2 inline'}/>
-            <img src={'/assets/images/logo-text.svg'} alt={'Instagram Logo Text'} className={'sm:invisible w-32 ml-2 mt-2 inline'}/>
+
+const Sidebar: React.FC<SidebarProps> = (props) => {
+
+      let [selectedItem, setSelectedItem] = useState(Navigation.FEED)
+
+      function onItemClick(id: Navigation) {
+            setSelectedItem(id)
+      }
+
+      function getIsSelected(id: Navigation) {
```

Comment:
totally an opinio but,
I like to keep name as short and unique as possible.
IMO isSelected would have been equally apt.
this is not wrong in any sense.

Reply:
True, in this particular case at least, _get_ is redundant.